src/twisted/_threads/test/test_team.py

```diff
@@ -14,7 +14,9 @@
 from twisted.python.failure import Failure
 from .. import IWorker, Team, createMemoryWorker, AlreadyQuit
 
-class ContextualWorker(proxyForInterface(IWorker, "_realWorker")):
+
+
+class ContextualWorker(proxyForInterface(IWorker, "_realWorker")):  # type: ignore[misc]  # noqa
```

Comment:
I don't know what to say.

Can't this be defined as the IWorker interface type ?

It is a proxy for that interface, so it should implement that interface.

Reply:
@adiroiban This code was added by @glyph in https://github.com/twisted/twisted/commit/b572e7e1ffa299f9751721405d5a5536995df85e#diff-cd7281783ad732cac003024589861371R18

Without this comment, mypy complains with:

```
src/twisted/_threads/test/test_team.py:17:24: error: Unsupported dynamic base class "proxyForInterface"  [misc]
    class ContextualWorker(proxyForInterface(IWorker, "_realWorker")):
```

Since `proxyForInterface` dynamically creates a class, this comment needs to be added to tell mypy to ignore this line, since mypy can only statically typecheck the class.

Reply:
Ok. In this case is ok to ignore all the proxyForInterface usage.

I guess that instead of creating a twisted pluging, is best if zope.interface could implement an `@delegator(ISomeThing, '_conextName')` decorator which will do the same thing as proxyForInterface.

I was using https://launchpad.net/lazr.delegates to get something similar but I think that this is something that has more to do with zope interface than zope components.